sre/roles/incidents/tasks/validate_ground_truth.yaml

```diff
@@ -52,6 +52,8 @@
   vars:
     aliases: "{{ incidents_ground_truth.aliases | flatten | unique }}"
     group_ids: "{{ incidents_ground_truth.groups | community.general.json_query('[*].id') }}"
+  when:
```

Comment:
See above comment about this field. If it is required, we should get rid of this.